src/Azure/AzureAppServicesIntegration/src/Microsoft.AspNetCore.AzureAppServicesIntegration.csproj

```diff
@@ -7,7 +7,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;azure;appservices</PackageTags>
-    <IsShippingPackage>true</IsShippingPackage>
+    <IsPackable>true</IsPackable>
```

Comment:
Why do a `$(IsShippingPackage)` -> `$(IsPackable)` switch in projects that aren't marked as `$(IsAspNetCoreApp)`? This project shouldn't have a ref/ project associated with it.